src/app/digital/actions/ports/InputPortChangeAction.ts

```diff
@@ -30,4 +30,8 @@ export class InputPortChangeAction extends PortChangeAction {
         return this;
     }
 
+    public getName(): string {
+        return "Inport Change";
```

Comment:
nit: typo

Reply:
I think this is intentional since the OutputPortChangeAction name is Outport Change